browser/react/components/App.js

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
-
 import '../../aframeComponents/scene-load';
+import '../../aframeComponents/aframe-hyperlink';
 import '../../aframeComponents/aframe-minecraft';
 
 export default function App (props) {
   return (
     <div style={{ width: '100%', height: '100%' }}>
       <a-scene id="scene" scene-load>
-        <a-entity geometry="primitive: sphere; radius: 1.25" position="0 1.25 -1" material="color: #EF2D5E"></a-entity>
-        <a-entity geometry="primitive: box; width: 1; height: 1; depth: 1" position="-1 0.5 1" material="color: #4CC3D9"></a-entity>
-        <a-entity geometry="primitive: cylinder; radius: 0.5; height: 1.5" position="1 0.75 1" material="color: #FFC65D"></a-entity>
-        <a-entity geometry="primitive: plane; width:4; height: 4" rotation="-90 0 0" material="color: #7BC8A4"></a-entity>
+        <a-assets>
```

Comment:
This actually can't be done. A-Frame errors out and says that the direct parent of the <a-assets> tag must be <a-scene>. I had it the way you suggest at first, but had to move everything into the main component to get A-Frame to load it. The best we could do would be to move all of the assets into an <Assets/> react component, but that would functionally be the same thing.